spec/dummy/config/initializers/sidekiq.rb

```diff
@@ -0,0 +1,13 @@
+sidekiq_connection = {
+  url: Settings.sidekiq.url,
```

Comment:
in dummy app you can explicite set sidekiq url and namespace - then `Settings` class, `air.yml` will not be needed